_includes/stylesheets.html

```diff
@@ -1,6 +1,6 @@
 
 <!-- Start stylesheets -->
-<link type="text/css" rel="stylesheet" href="{{ '/styles/preloader.css' | relative_url }}{% if site.github.build_revision %}?v={{ site.github.build_revision }}{% endif %}">
-<link type="text/css" rel="stylesheet" href="{{ '/styles/styles.css' | relative_url }}{% if site.github.build_revision %}?v={{ site.github.build_revision }}{% endif %}">
-<link type="text/css" rel="stylesheet" href="{{ '/styles/font-awesome.min.css' | relative_url }}{% if site.github.build_revision %}?v={{ site.github.build_revision }}{% endif %}">
+<link type="text/css" rel="stylesheet" href="{{ '/styles/preloader.css' | relative_url }}{% if site.github.build_revision %}?v={{ site.github.build_revision }}{% endif %}" data-proofer-ignore>
```

Comment:
Why do we need to add `data-proofer-ignore`?

Reply:
If we don't use `data-proofer-ignore`, html-proofer tries to find the stylesheets but can't find them as the url is unknown to proofer.